NAACCR/manual work/manual_work_08-Feb-23.sql

```diff
@@ -0,0 +1,171 @@
+--concept_relationship_manual_backup_2023_02_08;
+DO
+$body$
+    DECLARE
+        update text;
+    BEGIN
+        SELECT TO_CHAR(CURRENT_DATE, 'YYYY_MM_DD')
+        INTO update;
+        EXECUTE format('create table %I as select * from concept_relationship_manual',
+                       'concept_relationship_manual_backup_' || update);
+
+    END
+$body$;
+
+SELECT count(*) FROM concept_relationship_manual_backup_2023_02_08;
+
+--concept_manual_backup_2023_02_08;
+DO
+$body$
+    DECLARE
+        update text;
+    BEGIN
+        SELECT TO_CHAR(CURRENT_DATE, 'YYYY_MM_DD')
+        INTO update;
+        EXECUTE format('create table %I as select * from concept_manual',
+                       'concept_manual_backup_' || update);
+
+    END
+$body$;
+
+SELECT count(*) FROM concept_manual_backup_2023_02_08;
+
+--Upload https://docs.google.com/spreadsheets/d/1OLvyc4cSKHKNAo6jJy6EGIJUSOhaIRY4M1faG4y_bCY/edit#gid=0
+--It is slightly modified version of https://github.com/OHDSI/Vocabulary-v5.0/issues/740
+TRUNCATE concept_relationship_manual_refresh;
+CREATE TABLE concept_relationship_manual_refresh
+(
+    concept_code_1   varchar(50),
+    concept_code_2   varchar(50),
+    vocabulary_id_1  varchar(20),
+    vocabulary_id_2  varchar(20),
+    relationship_id  varchar(20),
+    valid_start_date DATE,
+    valid_end_date   DATE,
+    invalid_reason   varchar(1)
+)
+;
+
+UPDATE concept_relationship_manual_refresh
+SET valid_start_date= CURRENT_DATE-1;
+
+
+--CM population
+TRUNCATE concept_manual;
```

Comment:
@vladkorsik Why do we erase this table? Where do we store previously built mappings?

Reply:
@Alexdavv every time we are processing the NAACCR we create the backup for _manual tables and then populate _stage  with most recent data (e.g. from Community Contributors) stored in _refresh tables+ full _backup. The _refresh tables are created in order not to retain the bulky _manual NAACCR tables as the concept assemply and process is not Load Staged at all, but some kind of manual puzzle.

Reply:
@vladkorsik Can you create the GitHub issue to make it by our usual process in the future?

Reply:
let's discuss not in PR comments